ckanext/dataset_subscriptions/tests/actions/test_whatsapp_notifications.py

```diff
@@ -88,23 +88,29 @@ def create_user_with_resources(with_activity, with_notifications_enabled):
 @pytest.mark.usefixtures("with_plugins")
 def test_if_whatsapp_notifications_disabled():
     user = create_user_with_resources(with_activity=True, with_notifications_enabled=False)
-    phonenumber = whatsapp_notifications.get_phonenumber(user)
-    assert phonenumber is False
+    if whatsapp_notifications.whatsapp_notifications_enabled(user):
+        phonenumber = whatsapp_notifications.get_phonenumber(user)
+    else:
+        phonenumber = None
+    assert phonenumber is None
 
 
 @pytest.mark.usefixtures("clean_db")
 @pytest.mark.usefixtures("with_plugins")
 def test_if_whatsapp_notifications_enabled():
     user = create_user_with_resources(with_activity=True, with_notifications_enabled=True)
-    phonenumber = whatsapp_notifications.get_phonenumber(user)
+    if whatsapp_notifications.whatsapp_notifications_enabled(user):
+        phonenumber = whatsapp_notifications.get_phonenumber(user)
+    else:
+        phonenumber = None
```

Comment:
Like in my prev commnet. `with_notifications_enabled` = True/False is a perfect example for pytest parametrized test.
`get_phonemuber` should be tested separately. However, this method is so simple that maybe you could get it removed completely and just inline it's usage?

Reply:
I've decided to keep get_phonenumber as a separate function with dedicated test